src/components/Login/Login.jsx

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import styles from '../Login/Login.module.css';
+
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const loginEndpoint = 'http://localhost:4000/login';
+
+  const login = (event) => {
+    event.preventDefault();
+
+    axios
+      .post(loginEndpoint, {
+        name: event.target.elements.name.value,
+        email: event.target.elements.email.value,
+        password: event.target.elements.password.value,
+      })
+      .then((response) => {
+        localStorage['username'] = response.data.user.name;
+        localStorage['userToken'] = response.data.result;
+        navigate('/courses');
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  };
```

Comment:
Let's use `async/await` instead of promise chains, please, since chains is an old-fashioned approach